src/core/MOM_forcing_type.F90

```diff
@@ -2211,8 +2227,22 @@ subroutine mech_forcing_diags(forces, dt, G, time_end, diag, handles)
 
   integer :: i,j,is,ie,js,je
 
+  type(mech_forcing), pointer :: forces
+  integer :: turns
+
   call cpu_clock_begin(handles%id_clock_forcing)
 
+  ! NOTE: post_data expects data to be on the input index map, so any rotations
```

Comment:
This should have said "post_data expect data to be on the *rotated* index map...